how-to-team-desktop/issues.md

```diff
@@ -0,0 +1,85 @@
+# Issue Flow
+
+Below is a sample flow for resolving issues, from an issue being opened to closing. The *Triage* flow is intended to be done as soon as someone has the bandwidth to apply it to a newly opened issue. Once done, the issue may rest at the end of the flow until it is work for the current Pulse, then, the *Resolve* flow can be applied.
```

Comment:
If the team is happy to adopt the flow of having some low priority tickets marked `help-wanted` / `good-first-issue`, we could add a flow out of "Prioritize" that leaves it unassigned and open for a community contributor to pick up. 

We would need to document what the acceptance criteria are on the issue so that an external contributor can tackle it asynchronously / without too much input from us.

Reply:
Would that basically be the same as the "Resolve" flow, but without tagging as Jira and without assigning to a user? Maybe I could add a "Delegate" flow between Triage and Resolve that differentiates if a team member takes an issue or if a third-party contributor takes it?